wasmbinding/test/custom_message_test.go

```diff
@@ -0,0 +1,324 @@
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/CosmWasm/wasmd/x/wasm/keeper"
+	"github.com/CosmWasm/wasmvm/types"
+	host "github.com/cosmos/ibc-go/v3/modules/core/24-host"
+	"github.com/stretchr/testify/suite"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Nolus-Protocol/nolus-core/wasmbinding/bindings"
+
+	"github.com/neutron-org/neutron/app"
+	"github.com/neutron-org/neutron/testutil"
+	"github.com/neutron-org/neutron/wasmbinding"
+	feetypes "github.com/neutron-org/neutron/x/feerefunder/types"
+	icqkeeper "github.com/neutron-org/neutron/x/interchainqueries/keeper"
+	icqtypes "github.com/neutron-org/neutron/x/interchainqueries/types"
+	ictxkeeper "github.com/neutron-org/neutron/x/interchaintxs/keeper"
+)
+
+type CustomMessengerTestSuite struct {
+	testutil.IBCConnectionTestSuite
+	neutron         *app.App // TODO: nolus?
```

Comment:
Do yo think this pkg would continue to change a lot by neutron? My concern is that if there are a lot of changes it would be harder to find the diff and move it in our repo if we change all of the "neutron" naming

Reply:
Agreed, as we discussed, the test will remain based on neutron's app. 
I will remove all of the TODOs there, left them by mistake